composer.json

```diff
@@ -32,7 +32,8 @@
 	"require-dev": {
 		"leafo/lessphp": "~0.4.0",
 		"meenie/javascript-packer": "~1.1",
-		"phpunit/phpunit": "4.8.*"
+		"phpunit/phpunit": "4.8.*",
+		"tedivm/jshrink": "~1.1.0"
```

Comment:
"require-dev" is for "development purposes", running tests, etc

for "production", i.e end users or loaded via dependencies, it's just "require" block, it's noted in readme too to add `--no-dev`